salt/states/firewalld.py

```diff
@@ -75,40 +153,27 @@ def __virtual__():
     return (False, 'firewall-cmd is not available, firewalld is probably not installed.')
 
 
-def present(name,
```

Comment:
Is it necessary to rename this? If it's absolutely necessary we need to add documentation on this because we'll break existing states.